app/experimenter/static/rapid/types/experiment.ts

```diff
@@ -7,11 +7,18 @@ export enum ExperimentStatus {
   REJECTED = "Rejected",
 }
 
+export enum FirefoxChannel {
+  NIGHTLY = "Nightly",
+  BETA = "Beta",
+  RELEASE = "Release",
```

Comment:
Not sure if I'm understanding all the data flows and transforms correctly or if these are even related, but I vaguely recalled from https://github.com/mozilla/nimbus-shared/pull/99/files that the `browserSettings.update.channel == 'release'` and `normalized_channel = 'release'` channel names were both lowercased.
Not sure if that's in any way relevant or important, but figured worth calling out and highlighting my vast ignorance.

Reply:
Yeah, so the frontend contract is with the Experimenter V3 API, which defines the choice options as

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/constants.py#L194-L204

The contract between Experimenter and Firefox is encoded in the V4 API, which lower cases them here: 

https://github.com/mozilla/experimenter/pull/3144/files#diff-1146634a352f06a65bcaff252215820cR128

But an astute observation!